### PR TITLE
Topic/gatsby remark code repls add ramda

### PR DESCRIPTION
**Pull request by Undistraction**
Add support for loading code into [Ramda](http://ramdajs.com/repl/) REPL to gatsby-remark-code-repls.

Ref #3156

### CODE REVIEW

**Comment by Undistraction** [comment]
Hold fire on  this. Found an issue.

**Comment by Undistraction** [comment]
OK. It's good to go.

File: packages/gatsby-remark-code-repls/src/index.js [hunk 1]
```diff
@@ -130,6 +133,17 @@ module.exports = (
           node.children.length === 0 ? defaultText : node.children[0].value
 
         convertNodeToLink(node, text, href, target)
+      } else if (node.url.startsWith(PROTOCOL_RAMDA)) {
+        const filePath = getFilePath(node.url, PROTOCOL_RAMDA, directory)
+
+        verifyFile(filePath)
+
+        // Don't use `compress()` as the Ramda REPL won't understand the output. It uses URI to encode the code for its urls, so we do the same.
```
Comment (bvaughn) [inline packages/gatsby-remark-code-repls/src/index.js]: nit: Long line. Break into 2?

**Comment by KyleAMathews** [comment]
Thanks!

**Pull request merged**